.circleci/build.sh

```diff
@@ -70,6 +76,20 @@ elif [[ "$CIRCLE_JOB" == RELEASE_WITH_EXPENSIVE_TESTS ]]; then
     CMAKE_ARGS+=("-DGLOW_MODELS_DIR=$MODELS_DIR")
     CMAKE_ARGS+=("-DGLOW_WITH_OPENCL=OFF")
     CMAKE_ARGS+=("-DCMAKE_BUILD_TYPE=Release")
+elif [[ "$CIRCLE_JOB" == COVERAGE ]]; then
+    sudo apt-get install wget
+    ../utils/install_protobuf.sh
+    sudo apt-get install -y lcov libgoogle-glog-dev
+    sudo pip install awscli --upgrade
+    sudo apt-get install -qq g++-4.8
+    sudo update-alternatives --install /usr/bin/gcov gcov /usr/bin/gcov-4.8 90
+    CMAKE_ARGS=("CC=gcc-4.8 CXX=g++-4.8 
+          -DCMAKE_BUILD_TYPE=Debug -DGLOW_WITH_OPENCL=OFF -DGLOW_WITH_CPU=ON
+          -DLLVM_DIR=/usr/lib/llvm-7/cmake
+          -DCMAKE_CXX_FLAGS=-Werror
+          -DGLOW_USE_COVERAGE=ON")
+elif [[ "$CIRCLE_JOB" == CHECK_CLANG_FORMAT ]]; then
+    sudo apt-get install -y clang-format-8
```

Comment:
make sure the right version is used so that clang-format produces the same result (as in travis/locally)